auth/passport.js

```diff
@@ -0,0 +1,52 @@
+const passport = require("passport");
+const GoogleStrategy = require("passport-google-oauth").OAuth2Strategy;
+const User = require("../models/user");
+
+
+
+passport.use(
+    new GoogleStrategy(
+      {
+        clientID: process.env.GOOGLE_CLIENT_ID,
+        clientSecret: process.env.GOOGLE_SECRET,
+        callbackURL: process.env.GOOGLE_CALLBACK,
+      },
+      function (accessToken, refreshToken, profile, cb) {
+        User.findOne({ googleId: profile.id }, function (err, user) {
+          if (err) return cb(err);
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
          if (err) {return cb(err);}
```